pycqed/instrument_drivers/meta_instrument/qubit_objects/QuDev_transmon.py

```diff
@@ -1284,28 +1327,21 @@ def measure_randomized_benchmarking(
             det_func = self.int_avg_det
         MC.set_detector_function(det_func)
 
-        data_processing_pipe += [
-            # average data
-            {'node_type': 'average',
-             'num_bins': [len(cliffords)]*len(det_func.value_names),
-             'data_keys_in': det_func.value_names,
-             'data_keys_out': [e+'_avg' for e in det_func.value_names], **kw},
-            # std for each clifford sequence
-            {'node_type': 'get_std_deviation',
-             'num_bins': [len(cliffords)]*len(det_func.value_names),
-             'data_keys_in': det_func.value_names,
-             'data_keys_out': [e+'_std' for e in det_func.value_names], **kw},
-            # do SgQbRB
-            {'node_type': 'SingleQubitRBAnalysis',
-             'std_keys': [e+'_std' for e in det_func.value_names],
-             'data_keys_in': [e+'_avg' for e in det_func.value_names],
-             'meas_obj_name': self.name, **kw}]
-
-        # create sweep points
+        # create sweep points object
         sp = SweepPoints('nr_seeds', np.arange(nr_seeds), '', 'Nr. Seeds')
         sp.add_sweep_dimension()
         sp.add_sweep_parameter('cliffords', cliffords, '',
                                'Number of applied Cliffords, $m$')
+        # create analysis pipeline object
```

Comment:
I like this notation ! Would make sense to also have `pp.add_nodes(node1, node2, ...)`

Reply:
Yep, this is already implemented :)